_help/troubleshooting.md

```diff
@@ -325,3 +325,27 @@ Check your `ulimit -a`. Many systems have a 1024 open file descriptor limit by d
 ```
 
 Make sure to raise your ulimit. Example: `ulimit -n 16384`
+
+## Envoy Process High CPU Usage
+
+For larger clusters, the default configuration that comes with Istio
+refreshes the Envoy configuration every 1 second. This can cause high
+CPU usage, even when Envoy isn't doing anything. In order to bring the
+CPU usage down for larger deployments, increase the refresh interval for
+Envoy to something higher, like 30 seconds.
+
+```
+# increase the field rdsRefreshDelay in the mesh and defaultConfig section
+# set the refresh interval to 30s
+kubectl edit configmap -n istio-system istio
+
+# restart pilot and wait a few minutes
+kubectl delete pods -n istio-system -l istio=pilot
+```
+
+Also make sure to reinject the sidecar into all of your pods, as
```

Comment:
That shouldn't be necessary (the refresh rate after the first connection is controlled by pilot afaik)

Reply:
No, we need to re-inject here since some refresh rates are only defined at startup (limitation of v1 APIs).